tests/unittests/modules/test_network.py

```diff
@@ -0,0 +1,456 @@
+# -*- coding: utf-8 -*-
+
+# Import Python Libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+import os.path
+import socket
+
+import hubblestack.config
```

Comment:
It doesn't seem to find this package, I think you need to port this in https://github.com/saltstack/salt/tree/master/salt/config

Reply:
Yea I believe Paul is working on that according to the wiki.